manifests/init.pp

```diff
@@ -151,6 +151,10 @@
 #   Integer.  The integer value for the RabbitMQ prefetch attribute
 #   Default: 1
 #
+# [*rabbitmq_cluster*]
+#   Array. Rabbitmq Cluster configuration and connection information for one or more Cluster
```

Comment:
Would explicitly state that it is an `Array of hashes` to avoid confusion. I would also put in some validation to error with an explanation if it is not an array of hashes (having been bitten by this before).